src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/format/SelectResultSet.java

```diff
@@ -92,7 +93,7 @@ private void loadFromEsState(Query query) {
         String typeName = fetchTypeName(query);
         String[] fieldNames = fetchFieldsAsArray(query);
 
-        if (fieldNames.length == 0 && !fieldsSelectedOnAnotherTable(query))
+        if (isSimpleQuerySelectAll(query) || isJoinQuerySelectAll(query, fieldNames))
```

Comment:
please change to
```java
selectAll = isSimpleQuerySelectAll(query) || isJoinQuerySelectAll(query, fieldNames);
```

In general, let's keep consistent style and use curly braces even if the block will have 1 line.

Reply:
> please change to
> 
> ```java
> selectAll = isSimpleQuerySelectAll(query) || isJoinQuerySelectAll(query, fieldNames);
> ```
> 
> In general, let's keep consistent style and use curly braces even if the block will have 1 line.

Sure, will fix.